app/views/user_mailer/account_activation.text.erb

```diff
@@ -1,3 +1,3 @@
-<%= t "user_mailer.hi" %> <%= @user.name %>,
-<%= t "user_mailer.account_span" %>
+<%= t "activation.mailer.hi " %> <%= @user.name %>,
```

Comment:
You should use `params in I18n` for this code `<%= t "activation.mailer.hi " %> <%= @user.name %>,`